src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md

```diff
@@ -0,0 +1,150 @@
+
+Here's an example:
+
+```csharp
+#addin Cake.Email
+
+Task("SendEmail")
+    .Does(() =>
+{
+    try
+    {
+        var result = Email.Send(
+                senderName: "Bob Smith", 
+                senderAddress: "bob@example.com",
+                recipientName: "Jane Doe",
+                recipientAddress: "jane@example.com",
+                subject: "This is a test",
+                content: "<html><body>This is a test</body></html>",
+                sendAsHtml: true,
+                settings: new EmailSettings 
+                {
+                    SmtpHost = "smtp.gmail.com",
+                    Port = 587,
+                    EnableSsl = true,
+                    Username = "my_gmail_address@gmail.com",
+                    Password = "my_password"
+                }
+        );
+
+        if (result.Ok)
+        {
+            Information("Email succcessfully sent");
+        }
+        else
+        {
+            Error("Failed to send email: {0}", result.Error);
+        }
+    }
+    catch(Exception ex)
+    {
+        Error("{0}", ex);
+    }
+});
+```
+
+
+__Cake.SendGrid__
+
+The second addin allows you to send emails via a third-party Email Service Provider (ESP) called [SendGrid](https://sendgrid.com). The parameters you pass to the `SendGrid.SendEmail` alias are very similar to what we described earlier except that it accepts both the HTML and the plain text versions which will be sent in the same "multi-part" email, and also the only setting expected by SendGrid is your API Key.
+
+Here's an exmaple:
+
+```csharp
+#addin Cake.SendGrid
+
+var sendGridApiKey = EnvironmentVariable("SENDGRID_API_KEY");
+
+Task("SendEmail")
+    .Does(() =>
+{
+    try
+    {
+        var result = SendGrid.SendEmail(
+                senderName: "Bob Smith", 
+                senderAddress: "bob@example.com",
+                recipientName: "Jane Doe",
+                recipientAddress: "jane@example.com",
+                subject: "This is a test",
+                htmlContent: "<html><body>This is a test</body></html>",
+                textContent: "This is a test",
+                settings: new SendGridEmailSettings { ApiKey = sendGridApiKey }
+        );
+
+        if (result.Ok)
+        {
+            Information("Email succcessfully sent");
+        }
+        else
+        {
+            Error("Failed to send email: {0}", result.Error);
+        }
+    }
+    catch(Exception ex)
+    {
+        Error("{0}", ex);
+    }
+});
+```
+
+__Cake.CakeMail__
```

Comment:
Can you make this a link to the NuGet Package?

Also, can you add a link within the paragraph to the source repository?